### PR TITLE
update windows-defender-exploit-guard.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #3257
modified table to clarify requirements
added note about Identity & Threat Protection package

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/windows-defender-exploit-guard.md [hunk 1]
```diff
@@ -57,12 +57,15 @@ This section covers requirements for each feature in Windows Defender EG.
...
+| Attack surface reduction rules | ![not supported](./images/ball_empty.png) | ![not supported](./images/ball_empty.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+| Network protection | ![not supported](./images/ball_empty.png) | ![not supported](./images/ball_empty.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+| Controlled folder access | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+
+>[!NOTE]
+> [Identity & Threat Protection package](https://www.microsoft.com/microsoft-365/blog/2019/01/02/introducing-new-advanced-security-and-compliance-offerings-for-microsoft-365/), available for Microsoft 365 E3 customers, provides the same Windows Defender ATP capabilities as Enterprise E5 subscription.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/windows-defender-exploit-guard.md]:
```suggestion
>The [Identity & Threat Protection package](https://www.microsoft.com/microsoft-365/blog/2019/01/02/introducing-new-advanced-security-and-compliance-offerings-for-microsoft-365/), available for Microsoft 365 E3 customers, provides the same Windows Defender ATP capabilities as Enterprise E5 subscription.
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were applied, thank you!

File: windows/security/threat-protection/windows-defender-exploit-guard/windows-defender-exploit-guard.md [hunk 2]
```diff
@@ -57,12 +57,15 @@ This section covers requirements for each feature in Windows Defender EG.
...
+| Attack surface reduction rules | ![not supported](./images/ball_empty.png) | ![not supported](./images/ball_empty.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+| Network protection | ![not supported](./images/ball_empty.png) | ![not supported](./images/ball_empty.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+| Controlled folder access | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, limited reporting](./images/ball_50.png) | ![supported, full reporting](./images/ball_full.png) |
+
+>[!NOTE]
+> The [Identity & Threat Protection package](https://www.microsoft.com/microsoft-365/blog/2019/01/02/introducing-new-advanced-security-and-compliance-offerings-for-microsoft-365/), available for Microsoft 365 E3 customers, provides the same Windows Defender ATP capabilities as Enterprise E5 subscription.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/windows-defender-exploit-guard.md]:
```suggestion
> The [Identity & Threat Protection package](https://www.microsoft.com/microsoft-365/blog/2019/01/02/introducing-new-advanced-security-and-compliance-offerings-for-microsoft-365/), available for Microsoft 365 E3 customers, has the same Windows Defender ATP capabilities provided by Office 365 Enterprise E5.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix  ,
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@lizap @jaimeo @Justinha @DaniHalfin @mjcaparas @greg-lindsay - Can you please assist to check, approve and merge the content of this PR based on issue #3257?

Thank you.

**Comment by mypil** [comment]
@lizap @jaimeo @Justinha @DaniHalfin @mjcaparas @greg-lindsay - We would like to follow up on this. Can anyone of you please assist to check, approve and merge the content of this PR based on issue #3257?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr @lizap @jaimeo @Justinha @DaniHalfin @mjcaparas @greg-lindsay -  Can anyone of you please assist to check, approve and merge the content of this PR based on issue #3257?

Thank you.

**Pull request merged**